package.json

```diff
@@ -6,13 +6,18 @@
     "build:scss": "node-sass --output-style compressed -o dist/styles src/styles",
     "watch:scss": "onchange -i -v 'src/styles/*.scss' -- npm run build:scss",
     "lint:js": "eslint --fix src/js/*.js",
+    "transpile:js": "babel src -d dist",
```

Comment:
The structure of the „js” scripts becomes bloated with your changes. How about creating just build:js and watch:js? Also – you don't need to run linter and transpiler at the same time, i would suggest to run linter first, and after that run the transpiler. (npm run eslint --fix src/js/*.js && babel src -d dist)

Reply:
And one more thing – you're outputting js files to dist folder, but it should be `dist/js` and also – you should pass correct path into index.html <script> tag

Reply:
Babel creates index.js file in this path by default. If I set the output as a 'dist/js' it would be 'dist/js/js/index.js'